components/error_page/workspace_user_limit_reached.tsx

```diff
@@ -31,7 +42,11 @@ const WorkspaceUserLimitReached: React.FC = () => {
                         <div className='ErrorPage__maxFreeUsersReachedDescription'>
                             <FormattedMessage
                                 id='error.maxFreeUsersReached.description'
-                                defaultMessage='New users cannot be added to this workspace because it has reached the user limit for the free version of Mattermost Cloud. The system administrator has been notified'
+                                defaultMessage='New users cannot be added to this workspace because it has reached the user limit for the free version of Mattermost Cloud.'
+                            />
+                            <FormattedMessage
+                                id={notifyMessageId}
+                                defaultMessage=' '
```

Comment:
Very neat implementation here to update the message after the email was sent. However, UX-wise I believe it may look a bit distracting that it all of the sudden the texted appear without any user interaction. 

We should send the email silently in the background without changing the way the screen is showing the text.